pallets/connectors/src/lib.rs

```diff
@@ -686,6 +686,20 @@ pub mod pallet {
 
 			Ok(())
 		}
+		#[pallet::weight(10_000)]
+		#[pallet::call_index(10)]
+		pub fn schedule_rely(
+			origin: OriginFor<T>,
+			usr: DomainAddress,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+	
+			T::OutboundQueue::submit(
```

Comment:
and here we should ensure that the the `DomainAddress` is a `DomainAddress::EVM`, i.e, this should fail with an error when it's `DomainAddress::Centrifuge` since we don't want to be sending this message to the chain itself.